doc/api/crypto.md

```diff
@@ -1138,8 +1141,18 @@ changes:
 -->
 * {string}
 
-For asymmetric keys, this property represents the type of the embedded key
-(`'rsa'`, `'dsa'`, `'ec'`, `'ed25519'`, `'ed448'`, `'x25519'` or `'x448'`).
+For asymmetric keys, this property represents the type of the embedded key. This
+version of Node.js supports the following asymmetric key types:
```

Comment:
I'd just say "Supported key types are:" -- all the docs refer to "this version of node.js", it doesn't need to be said.